### DISCUSSION TITLE
NavigationView not working for skia

### DISCUSSION THREAD

**Discussion by rohan215** [unoplatform/uno#10492] [Q&A]
I'm trying to do basic stuff
`<NavigationView></NavigationView>` but on skia I get a error
`fail: Uno.UI.DataBinding.BindingPropertyHelper[0]
      The [AutomationProperties] property getter does not exist on type [Microsoft.UI.Xaml.Controls.NavigationView]
fail: Windows.ApplicationModel.Core.CoreApplicationViewTitleBar[0]
      The member event TypedEventHandler<CoreApplicationViewTitleBar, object> CoreApplicationViewTitleBar.LayoutMetricsChanged is not implemented in Uno.
fail: Windows.ApplicationModel.Core.CoreApplicationViewTitleBar[0]
      The member event TypedEventHandler<CoreApplicationViewTitleBar, object> CoreApplicationViewTitleBar.IsVisibleChanged is not implemented in Uno.
fail: Microsoft.UI.Xaml.UIElement[0]
      The member event TypedEventHandler<UIElement, AccessKeyInvokedEventArgs> UIElement.AccessKeyInvoked is not implemented in Uno.
fail: Windows.ApplicationModel.Core.CoreApplicationViewTitleBar[0]
      The member bool CoreApplicationViewTitleBar.ExtendViewIntoTitleBar is not implemented in Uno.
fail: Windows.ApplicationModel.Core.CoreApplicationViewTitleBar[0]
      The member bool CoreApplicationViewTitleBar.Height is not implemented in Uno.
fail: Uno.UI.Dispatching.CoreDispatcher[0]`
Any ideas why?

**Comment by jeromelaban** [comment]
Thanks for the report. Let's track in https://github.com/unoplatform/uno/issues/10494

**Reply by rohan215** [reply]
That was me on discord. Got it working now
